images/varnish/Dockerfile

```diff
@@ -2,7 +2,6 @@ ARG IMAGE_REPO
 FROM ${IMAGE_REPO:-lagoon}/commons as commons
 FROM alpine
 
-MAINTAINER amazee.io
```

Comment:
why is this removed?

Reply:
Because the MAINTAINER flags are deprected. We can add it via a label. Can create a seperate PR for that and go trough our images.


https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Reply:
i'll reverted that change and we can do that in a seperate pr.